Crash when PiP not supported

### Steps to reproduce

Just play any video

### Expected behavior

Play the video

### Actual behavior

App closes

### LibreTube version

0.15.0

### Android version

Android 9

### Other details

`java.lang.IllegalStateException: setPictureInPictureParams: Device doesn't support picture-in-picture mode.
 at android.os.Parcel.createException(Parcel.java:1958)
 at android.os.Parcel.readException(Parcel.java:1918)
 at android.os.Parcel.readException(Parcel.java:1868)
 at android.app.IActivityManager$Stub$Proxy.setPictureInPictureParams(IActivityManager.java:8997)
 at android.app.Activity.setPictureInPictureParams(Activity.java:2197)
 at retrofit2.Platform$$ExternalSyntheticApiModelOutline1.m(SourceFile:18)
 at com.github.libretube.ui.fragments.PlayerFragment.onDestroy(SourceFile:46)
 at androidx.fragment.app.FragmentStateManager.destroy(SourceFile:150)
 at androidx.fragment.app.FragmentStateManager.moveToExpectedState(SourceFile:312)
 at androidx.fragment.app.SpecialEffectsController$FragmentStateManagerOperation.complete(SourceFile:64)
 at androidx.appcompat.app.AppCompatDelegateImpl$AutoNightModeManager.completeSpecialEffect(SourceFile:23)
 at androidx.fragment.app.DefaultSpecialEffectsController.executeOperations(SourceFile:1865)
 at androidx.fragment.app.SpecialEffectsController.executePendingOperations(SourceFile:159)
 at androidx.fragment.app.FragmentManager.executeOpsTogether(SourceFile:1151)
 at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(SourceFile:82)
 at androidx.fragment.app.FragmentManager.execSingleAction(SourceFile:32)
 at com.github.libretube.helpers.NavigationHelper.navigateVideo$default(SourceFile:186)
 at com.github.libretube.ui.adapters.VideosAdapter$$ExternalSyntheticLambda0.onClick(SourceFile:70)
 at android.view.View.performClick(View.java:6601)
 at android.view.View.performClickInternal(View.java:6578)
 at android.view.View.access$3100(View.java:778)
 at android.view.View$PerformClick.run(View.java:25897)
 at android.os.Handler.handleCallback(Handler.java:873)
 at android.os.Handler.dispatchMessage(Handler.java:99)
 at android.os.Looper.loop(Looper.java:193)
 at android.app.ActivityThread.main(ActivityThread.java:6669)
 at java.lang.reflect.Method.invoke(Native Method)
 at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: android.os.RemoteException: Remote stack trace:
 at com.android.server.am.ActivityManagerService.ensureValidPictureInPictureActivityParamsLocked(ActivityManagerService.java:9152)
 at com.android.server.am.ActivityManagerService.setPictureInPictureParams(ActivityManagerService.java:9101)
 at android.app.IActivityManager$Stub.onTransact(IActivityManager.java:3027)
 at com.android.server.am.ActivityManagerService.onTransact(ActivityManagerService.java:3390)
 at android.os.Binder.execTransact(Binder.java:731)`

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

--- a/app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt
+++ b/app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt
@@ -622,10 +622,12 @@ class PlayerFragment : Fragment(), OnlinePlayerOptions {
 
         // disable the auto PiP mode for SDK >= 32
         exoPlayer.pause()
-        PictureInPictureCompat.setPictureInPictureParams(
-            requireActivity(),
-            pipParams
-        )
+        if (PlayerHelper.pipEnabled) {
+            PictureInPictureCompat.setPictureInPictureParams(
+                requireActivity(),
+                pipParams
+            )
+        }
 
         handler.removeCallbacksAndMessages(null)
 

